hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/decommissioningSummary/decommissioningSummary.tsx

```diff
@@ -0,0 +1,114 @@
+
+type DecommisioningSummaryProps = {
+  uuid: string;
+}
+
+type DecommisioningSummaryState = {
+  loading: boolean;
+  summaryData: SummaryData | Record<string, unknown>;
+};
+
+const DecommissionSummary: React.FC<DecommisioningSummaryProps> = ({
+  uuid = ''
+}) => {
+  const [state, setState] = React.useState<DecommisioningSummaryState>({
+    summaryData: {},
+    loading: false
+  });
+  const cancelSignal = React.useRef<AbortController>();
+
+  async function fetchDecommissionSummary(selectedUuid: string) {
+    setState({
+      ...state,
+      loading: true
+    });
+    try {
+      const { request, controller } = AxiosGetHelper(
+        `/api/v1/datanodes/decommission/info/datanode?uuid=${selectedUuid}`,
+        cancelSignal.current
+      );
+      cancelSignal.current = controller;
+      const datanodesInfoResponse = await request;
+      setState({
+        ...state,
+        loading: false,
+        summaryData: datanodesInfoResponse?.data?.DatanodesDecommissionInfo[0] ?? {}
+      });
+    } catch (error) {
+      setState({
+        ...state,
+        loading: false,
+        summaryData: {}
+      });
+      showDataFetchError((error as AxiosError).toString());
+    }
+  }
+
+  useEffect(() => {
+    fetchDecommissionSummary(uuid);
+    return (() => {
+      cancelRequests([cancelSignal.current!]);
+    })
+  }, []);
+
+  let content = (
+    <Result
+      status='error'
+      title='Unable to fetch Decommission Summary data'
+      className='decommission-summary-result'/>
+  );
+
+  const { summaryData } = state;
+  if (summaryData?.datanodeDetails && summaryData?.metrics && summaryData?.containers) {
+    const {
+      datanodeDetails: { uuid, networkLocation, ipAddress, hostName },
+      containers: { UnderReplicated, UnClosed },
+      metrics: { decommissionStartTime, numOfUnclosedPipelines, numOfUnclosedContainers, numOfUnderReplicatedContainers }
+    } = summaryData as SummaryData;
+    content = (
+      <Descriptions size="small" bordered column={1} title={`Decommission Status: DECOMMISSIONING`}>
+        <Descriptions.Item label="Datanode"> <b>{uuid}</b></Descriptions.Item>
+        <Descriptions.Item label="Location">({networkLocation}/{ipAddress}/{hostName})</Descriptions.Item>
+        <Descriptions.Item label="Decommissioning Started at">{decommissionStartTime}</Descriptions.Item>
+        <Descriptions.Item label="No. of Unclosed Pipelines">{numOfUnclosedPipelines}</Descriptions.Item>
+        <Descriptions.Item label="No. of Unclosed Containers">{numOfUnclosedContainers}</Descriptions.Item>
+        <Descriptions.Item label="No. of Under-Replicated Containers">{numOfUnderReplicatedContainers}</Descriptions.Item>
+        <Descriptions.Item label="Under-Replicated">{UnderReplicated}</Descriptions.Item>
+        <Descriptions.Item label="Unclosed">{UnClosed}</Descriptions.Item>
+      </Descriptions>
+    );
+  }
+  // Need to check summarydata is not empty 
```

Comment:
This comment seems not in sync what code is doing actually. Can we make it more meaningful ?

Reply:
Sorry, this was a previous comment which is not applicable after cleanup, removed in latest commit